### ISSUE TITLE
Make domains & subdomains lowercase

### ISSUE THREAD

**Issue by JPL1** [badcodeltd/safecms#32]
In 0.0.6 it is possible to create subdomains with uppercase letters. These are not found by Peruse / Safe Browser. Subdomains should be made lowercase before publishing.

**Comment by happybeing** [comment]
I regard this as an API issue. The API can't enforce lowercase (or anything else) at creation, but flagging it will help it get documented. What do you think? 

Same goes for unusual character sets - what does Peruse require is an important wisdom?

**Comment by JPL1** [comment]
You're right of course. I'll raise this issue on the Peruse GitHub. Guessing that it might take a while to implement though and in the meantime setting input to lowercase in the domain and subdomain fields would be a quick fix and cover most cases.
